### PR TITLE
Move LOG() into utils module

### PR DESCRIPTION
**Pull request by jwerle**
Fixes https://github.com/GoogleChrome/omnitone/issues/14
cc @agrathwohl

### CODE REVIEW

File: src/omnitone.js [hunk 1]
```diff
@@ -31,26 +31,20 @@ var FOARotator = require('./foa-rotator.js');
 var FOAPhaseMatchedFilter = require('./foa-phase-matched-filter.js');
 var FOAVirtualSpeaker = require('./foa-virtual-speaker.js');
 var FOADecoder = require('./foa-decoder.js');
+var utils = require('./utils');
```
Comment (hoch) [inline src/omnitone.js]: Perhaps we can make them consistent?

`var Utils = require('./utils.js');`

File: src/omnitone.js [hunk 2]
```diff
@@ -31,26 +31,20 @@ var FOARotator = require('./foa-rotator.js');
...
-    'color: #AAA'
-  ]);
-};
-
+/**
+ * Omnitone library logging function.
+ * @type {Function}
+ * @param {any} Message to be printed out.
+ */
+Omnitone.LOG = utils.LOG;
```
Comment (hoch) [inline src/omnitone.js]: `Utils`

File: src/omnitone.js [hunk 3]
```diff
@@ -67,7 +61,6 @@ Omnitone.loadAudioBuffers = function (context, speakerData) {
   });
 };
 
-
```
Comment (hoch) [inline src/omnitone.js]: I keep two blank lines between functions.

File: src/utils.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+ * limitations under the License.
+ */
+
+/**
+ * @fileOverview Omnitone library common utilities.
+ */
+
+'use strict';
+
+// Omnitone library-wide log utility.
```
Comment (hoch) [inline src/utils.js]: I think it's best to follow JSDoc style. Please refer the style of other files.

```
/**
 * comments...
 */
```

**Comment by hoch** [comment]
Thanks so much for your contribution. Can you make some edits that I suggested?

File: src/omnitone.js (hunk 1 above)
Comment (jwerle) [inline src/omnitone.js]: Agreed :)

File: src/utils.js (hunk 4 above)
Comment (jwerle) [inline src/utils.js]: This was copy-pasta from `omnitone.js`. I'll correct now!

**Comment by jwerle** [comment]
@hoch done! I've rebased/squashed commits for a cleaner merge.

File: src/foa-decoder.js [hunk 5]
```diff
@@ -26,7 +26,7 @@ var FOARotator = require('./foa-rotator.js');
 var FOAPhaseMatchedFilter = require('./foa-phase-matched-filter.js');
 var FOAVirtualSpeaker = require('./foa-virtual-speaker.js');
 var FOASpeakerData = require('./foa-speaker-data.js');
-
+var Utils = require('./utils');
```
Comment (hoch) [inline src/foa-decoder.js]: Can we add '.js' at the end of file path? I know it's not required, but I want to keep them consistent.

File: src/audiobuffer-manager.js [hunk 6]
```diff
@@ -19,6 +19,8 @@
 
 'use strict';
 
+var Utils = require('./utils');
```
Comment (hoch) [inline src/audiobuffer-manager.js]: Can we add '.js' at the end of file path? I know it's not required, but I want to keep them consistent.

File: src/foa-phase-matched-filter.js [hunk 7]
```diff
@@ -21,6 +21,8 @@
 
 'use strict';
 
+var Utils = require('./utils');
```
Comment (hoch) [inline src/foa-phase-matched-filter.js]: Ditto.

File: src/omnitone.js [hunk 8]
```diff
@@ -31,26 +31,20 @@ var FOARotator = require('./foa-rotator.js');
 var FOAPhaseMatchedFilter = require('./foa-phase-matched-filter.js');
 var FOAVirtualSpeaker = require('./foa-virtual-speaker.js');
 var FOADecoder = require('./foa-decoder.js');
+var Utils = require('./utils');
```
Comment (hoch) [inline src/omnitone.js]: Ditto.

**Comment by hoch** [comment]
@jwerle Sorry, just one more nit. Can you take a look?

**Comment by hoch** [comment]
Also did you test the minified library with test files? A local test is good enough, I think.

https://github.com/GoogleChrome/omnitone/tree/master/test

**Comment by jwerle** [comment]
@hoch done, and yes I did

File: build/omnitone.js [hunk 9]
```diff
@@ -230,6 +215,8 @@ return /******/ (function(modules) { // webpackBootstrap
 
 	'use strict';
 
+	var utils = __webpack_require__(3)
```
Comment (hoch) [inline build/omnitone.js]: Hmm. Not sure why this still shows `utils`. Any idea?

Comment (jwerle) [inline build/omnitone.js]: oops! I was running `npm run build-all` and forgetting to run `npm run build`

**Comment by hoch** [comment]
Awesome! Merged.

**Comment by jwerle** [comment]
@hoch thanks !!

**Comment by hoch** [comment]
@jwerle Thank YOU! 👍

**Comment by jwerle** [comment]
@hoch We will be adding spatial media support in https://github.com/littlstar/axis thanks to this PR :)

**Comment by hoch** [comment]
@jwerle A wonderful project! Will definitely check out soon. :)

**Comment by jwerle** [comment]
@hoch great!!

**Pull request merged**